app/onboarding/views/final.py

```diff
@@ -0,0 +1,26 @@
+from app.extensions import limiter
+from app.onboarding.base import onboarding_bp
+from app.email_utils import send_test_email_alias
+from flask import render_template, request, flash
+from flask_login import current_user, login_required
+from flask_wtf import FlaskForm
+from wtforms import StringField, validators
+
+
+class SendEmailForm(FlaskForm):
+    email = StringField("Email", validators=[validators.DataRequired()])
+
+
+@onboarding_bp.route("/final", methods=["GET", "POST"])
+@login_required
+@limiter.limit("10/minute")
+def final():
+    form = SendEmailForm(request.form)
+    if form.validate_on_submit():
+        send_test_email_alias(form.email.data, current_user.name)
```

Comment:
That's a good idea to use the existing `send_test_email_alias` here 👌. I think we need to update the wording in the email to remove the "If you have any questions, feel free to reply to this email." part as now all emails are sent from a no-reply address.

Reply:
We could change the line by

```
If you have any questions, feel free to contact us either by sending an e-mail to <a href="mailto:hi@simplelogin.io">hi@simplelogin.io</a) or through our <a href="/dashboard/support">support page</a>
```

Reply:
Pushed an initial implementation. Let me know what you think.

Reply:
I would remove the part about sending us and email and reference them to use our support channel.

Reply:
Replaced it with

HTML:
```
{{ render_text('If you have any questions, feel free to contact us through our <a href="' + support_page +'">support page</a>') }}
```

TXT:
```
If you have any questions, feel free to contact us through our support page: {{support_page}}
```